djaoapp/templates/saas/_filter.html

```diff
@@ -47,10 +47,5 @@
     </uiv-dropdown>
 </div>
 <div class="col">
-    <div class="input-group input-group-sm">
-      <div class="input-group-prepend">
-        <span class="input-group-text" id="filter-inp">Match</span>
-      </div>
-      <input class="form-control" type="text" v-model="params.q" @input="filterList" />
-    </div>
+    {% include "saas/_filter_input.html" %}
```

Comment:
`_filter_match.html` or `_filter_by_text.html` might be better names. `_filter_input.html` is a little confusing when a `_filter.html` already exists.